config/names/nightmares.txt

```diff
@@ -0,0 +1,27 @@
+a dead skrell
+a dead unathi
+a dead tajaran
+a dead vulpkanin
```

Comment:
_something space-racist_

Reply:
`a dead meme`

Reply:
You forgot fish folk

Reply:
"a dead skrell"

Reply:
I meant @LittleBigKid2000

Reply:
No one is racist against skrell because they aren't furry.

Reply:
Neither are lizards. They're scaly. Skrell are slimy. They might as well be mermaids

Reply:
Skrell are tentacle frogs.

Reply:
[Tentacle frog?](http://www.meh.ro/wp-content/uploads/2013/01/meh.ro10870.jpg)